docs/ja/intro-to-loom.md

```diff
@@ -14,26 +14,26 @@ In short, you can think of Loom Network as EOS on top of Ethereum.
 
 Our core product is an SDK that allows developers to quickly build their own blockchains without having to understand blockchain infrastructure. Think of it like a “build your own blockchain” generator.
 
-## What is PlasmaChain?
+## What is Basechain?
 
 So, the Loom SDK allows for developers to spin up an unlimited number of sidechains to scale their dapps as needed — a crucial piece in our long-term vision of blockchain scalability, and one that is now being used by multiple games and dapps in production.
 
-But PlasmaChain will be the central hub that all these chains connect to as their bridge to Ethereum, for interchain communication with other major blockchains (such as Cosmos, EOS, and Tron), and as a shared liquidity pool for tokenized assets.
+But Basechain will be the central hub that all these chains connect to as their bridge to Ethereum, for interchain communication with other major blockchains (such as Cosmos, EOS, and Tron), and as a shared liquidity pool for tokenized assets.
 
-Thus, our company’s primary objectives over the next year are all related to growing PlasmaChain to become the “universal Layer 2” for high performance dapps.
+Thus, our company’s primary objectives over the next year are all related to growing Basechain to become the “universal Layer 2” for high performance dapps.
```

Comment:
Thus, our company’s primary objectives over the next year are all related to growing Basechain to become the “universal Layer 2” for high performance dapps. 
--> 
Thus, our company’s primary objectives are all related to growing Basechain to become the universal multichain interop platform for high-performance dapps.